### PR TITLE
Bckend/183 post comments

### PR DESCRIPTION
**Pull request by JoePotentier**

### CODE REVIEW

File: distributedsocialnetwork/api/views.py [hunk 1]
```diff
@@ -397,6 +397,28 @@ def post(self, request, pk):
...
+                        "query": "addComment",
+                        "success": False,
+                        "message": "This node is not authorized to post comments on this server."
+                    }, status=status.HTTP_403_FORBIDDEN)
+                elif (len(Author.objects.filter(id=comment["author"]["id"])) == 0):
+                    # Node is authorized but auther does now exit yet
+                    node = list(Node.objects.filter(
+                        hostname__contains=comment["author"]["host"]))[0]
+                    comment["author"]['displayName'] = comment["author"]['displayName'] + \
+                        " (" + node.server_username + ")"
```
Comment (ColonelSanders21) [inline distributedsocialnetwork/api/views.py:413]: Don't forget to set the URL appropriately. As of now if you make a new user his link will not work.

**Review by ColonelSanders21 — CHANGES_REQUESTED** [review]
This seems to work very well, we just need to be setting the urls of new users to be consistent with the rest of our foreign saved users.

File: distributedsocialnetwork/api/views.py (hunk 1 above)
Comment (JoePotentier) [reply distributedsocialnetwork/api/views.py:413]: Sounds good, my eyes were hurting so bad from staring at the screen that I could barely figure out what was going on anymore :')

Comment (JoePotentier) [reply distributedsocialnetwork/api/views.py:413]: Added change, please review.

**Pull request merged**